content/variables/git.md

```diff
@@ -19,6 +19,8 @@ toc: false
 wip: false
 ---
 
+This variable shows metada about a specific piece of content. It is useful for times when you are using Git and Hugo to manage a project or in a multi-author setting, for example. It will not be useful as a cache busting method.
```

Comment:
I don't like this addition. I think this "using this as a cache busting method" was something @brunoamaral thought about and not something the Common Joe will think about. We cannot fill the docs with "this is not that ...", so unless more people chime in with the same confusion, we're keeping this as is. So, thanks for this, but we'll wait.